jest.config.json

```diff
@@ -4,7 +4,8 @@
   "collectCoverage": false,
   "collectCoverageFrom": ["src/**/*.{ts,tsx,js,jsx}", "!src/**/*.d.ts"],
   "moduleNameMapper": {
-    "\\.(css|gif)$": "jest-transform-stub"
+    "\\.(css|gif)$": "jest-transform-stub",
+    "\\.svg": "<rootDir>/__mocks__/svg.js"
```

Comment:
let's move to `src/__tests__/__mocks__/svg.js`

Reply:
Jest complains that svg.js isnt a test if we have mocks in there

Reply:
we can tell `jest` to ignore these 
```
  "moduleNameMapper": {
    "\\.(css|gif)$": "jest-transform-stub",
    "\\.svg": "<rootDir>/src/__tests__/__mocks__/svg.js"
  },
  "moduleDirectories": ["node_modules", "assets"],
  "testPathIgnorePatterns": ["src/__tests__/__mocks__"]
```

Reply:
Added ignore for mocks folder